oddt/surface.py

```diff
@@ -0,0 +1,128 @@
+"""This module generates, plots, and does computation with molecular surfaces.
+"""
+
+import numpy as np
+from skimage import measure
+from skimage.morphology import ball, binary_closing
+from scipy.spatial import cKDTree
+
+
+def generate_surface_marching_cubes(molecule, remove_hoh=False, scaling=1,
+                                    probe_radius=1.4):
+    """Generates a molecular surface mesh using the marching_cubes_lewiner
+    method from scikit-image.
+
+    Parameters
+    ----------
+    molecule : oddt.toolkit.Molecule object
+        Molecule for which the surface will be generated
+
+    remove_hoh : bool (default = False)
```

Comment:
User needs to know that `molecule.protein` must be set to True in order to use this option. Add this requirement to docstring.